jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -40,6 +40,40 @@
   atexit.register(correctness_stats.pprint_all_limitations,
                   output_file, template_file)
 
+
+def _make_tf_args(args):
+  def _convert_if_bfloat16(v):
```

Comment:
These utils are not tied to ConvertAndCompare, or to the test class. It makes more sense to have them out here for reuse, I think.